specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/standardAssignments.json

```diff
@@ -0,0 +1,389 @@
+            "$ref": "#/parameters/StandardAssignmentName"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the standard assignment.",
+            "schema": {
+              "$ref": "#/definitions/StandardAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/standardAssignments": {
+      "get": {
+        "x-ms-examples": {
+          "List standard assignments": {
+            "$ref": "./examples/StandardAssignments/ListStandardAssignments.json"
+          }
+        },
+        "tags": [
+          "StandardAssignments"
+        ],
+        "description": "Get a list of all relevant standard assignments over a scope",
+        "operationId": "StandardAssignments_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StandardAssignmentsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StandardAssignmentsList": {
+      "type": "object",
+      "description": "Page of a standard assignment list",
+      "properties": {
+        "value": {
+          "description": "Collection of standardAssignments in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StandardAssignment"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "StandardAssignment": {
+      "type": "object",
+      "description": "Security Assignment on a resource group over a given scope",
+      "properties": {
+        "properties": {
+          "description": "Properties of a standard assignments assignment",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StandardAssignmentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
```

Comment:
>"../../../common/v1/types.json#/definitions/Resource" [](http://example.com/codeflow?start=18&length=53)

consider using "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource" which include "systemData":[https://github.com/Azure/azure-openapi-validator/blob/main/docs/required-read-only-system-data.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/required-read-only-system-data.md)

Reply:
It's not a tracked resource